docs/Conceptual/00 - Introduction.md

```diff
@@ -0,0 +1,195 @@
+# Multi-tenanted solution architecture
+In this document, we’re going to discuss concepts in multi-tenanted solution architecture.
+
+We will talk about
+- what multi-tenanted means
+- when to choose a multi-tenanted architecture
+- tenancy as a hierarchy
+- the management plane and the solution plane
+- how tenancy impacts application code and resource choices
+- how Corvus.Tenancy and Marain.Tenancy help with building multi-tenanted services.
+
+## What does multi-tenanted mean? 
+Let’s start by thinking about the distinction between *multi-tenant* and *multi-user*.
+
+The chief problem with giving multiple users access to a computer is that we need a fair way of ensuring that they don’t tread on each other’s toes.
+
+Initially, people used to book time on the computer. They would get exclusive access to a giant room of glowing valves, for a specific period of time. Then get physically booted out.
+
+By the 1970s, the slightly confusingly named [time sharing](https://en.wikipedia.org/wiki/Time-sharing) model predominated. Multiple users could access a single computer at the same time, sharing the resources.
+
+The security and fair-shares implications of this ultimately led to the development of isolated process models, virtual machines, containers, and the cloud/edge computing model we know today. 
+
+As part of this shift, the line between “computer user” and “solution user” has also blurred.
+
+In a modern web application, for example, there is frequently a single logical solution serving hundreds, thousands, or millions of concurrent users.
+
+That solution is likely implemented using many compute and storage resources, distributed across various physical and virtual machines, but it is conceptually *one solution, lots of users*. To developer and user alike, it looks like a single set of resources, serving many users. Those users may work together in logical groups (like teams, or organizations) but they are fundamentally sharing a single solution.
+
+Multi-tenancy, on the other hand, can be thought of as *lots of solutions, lots of users*. To the *owner* of the multi-tenanted solution provider, it looks conceptually like *many isolated child solutions* (or *tenants*) consuming the resources it provides, each of which independently serves many users. To any individual tenant, their “tenancy” looks like a single set of resources that they use to serve their users, and no-one else’s.
+
+![multi-user v. multi tenant](multi-user v multi-tenant.png)
```

Comment:
This still isn't rendering for me. And it's not just the diff preview - I went and checked how it looked browsing this branch normally in GitHub.

My theory is that it has something to do with the spaces. Maybe %20 would work—I think that's meant to be a URL (relative or absolute) so spaces aren't technically legal. Some markdown flavours support additional info following the URL in the parens, so spaces may well be a delimited there.

Reply:
The spec for a GFMD link destination is at https://github.github.com/gfm/#link-destination

It might work if you wrap the link in angle brackets. It looks like link destinations (which would include image URLs) come in two forms: `<`/`>`-delimited, and non-delimited. the non-delimited form is there to avoid clutter, but imposes some restrictions. The spec starts thus:

> a nonempty sequence of characters that does not start with <, **does not include ASCII space**

Since your image file names in this doc all contain spaces, either you need to rename them, or you need to use `<` and `>` to delimit the link.